megamek/src/megamek/client/ui/swing/util/UIUtil.java

```diff
@@ -141,6 +141,40 @@ public static String spanCSS(String cssClass, String content) {
         return "<span class=\"" + cssClass + "\">" + content + "</span>";
     }
 
+    /**
+     * Returns an anchor with opening and closing tag and the
+     * content in between.
+     * Used for internal navigation in a document
+     *
+     * @param anchorId the name of the anchor
+     * @param content  the text to go in the span
+     * @return the complete anchor block
+     */
+    public static String anchor(String anchorId, String content) {
+        return "<a name=\"" + anchorId + "\">" + content + "</a>";

```

Comment:
most recently you have to use `id` in the anchor for it to work, however swing didnt really followed the advancements in HTML standards so it still relies on using `name` for internal anchors.